Cargo.toml

```diff
@@ -23,25 +24,26 @@ k8s-openapi = { version = "0.25.0", default-features = false, features = [
 ] }
 lazy_static = "1.4.0"
 pem = "3"
-policy-evaluator = { git = "https://github.com/kubewarden/policy-evaluator", tag = "v0.25.2" }
-prettytable-rs = "^0.10"
-regex = "1"
-rustls-pki-types = { version = "1", features = ["alloc"] }
-semver = { version = "1.0.22", features = ["serde"] }
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
-serde_yaml = "0.9.34"
-tar = "0.4.40"
-termimad = "0.33.0"
-thiserror = "2.0"
-time = "0.3.36"
-tiny-bench = "0.4"
-tokio = { version = "^1.42.0", features = ["full"] }
-tracing = "0.1"
+#policy-evaluator = { git = "https://github.com/kubewarden/policy-evaluator", tag = "v0.25.2" }
+policy-evaluator   = { git = "https://github.com/flavio/policy-evaluator", branch = "kwctl-policy-group-changes" }
```

Comment:
Just a comment here to make more visible that we need to change here after release the policy-evaluator